### ISSUE TITLE
Add support for 5GHz ac channels

### ISSUE THREAD

**Issue by JuniorJPDJ** [chrisk44/Hijacker#35]
They are missing ;)

**Comment by chrisk44** [comment]
This needs to be supported by the wireless adapter and the firmware.
However, airodump-ng is, by default, hopping on 2.4Ghz channels and yes, I haven't set it to do anything else.

The project is going to be on hold for some time, but it may merge with nexmon and use a completely different way of capturing packets. As I'm not sure exactly how it will work, I can't guarantee support for 5Ghz, but I'll see what I can do.
I will provide an update as soon as there is one.

**Comment by JuniorJPDJ** [comment]
I mean just in GUI.
My chipset and nexmon's airodump installation works with that, but I can't select channel in GUI when using your app.

**Comment by chrisk44** [comment]
Yes, that's what I said. I'm not making any changes to the app right now, since I'm probably going to change everything. For now, it stays as it is.

**Comment by chrisk44** [comment]
Try the latest pre-release (v1.5-beta.2).

**Comment by JuniorJPDJ** [comment]
I will check, thanks

**Comment by schinfo** [comment]
Can you include channels >= CH 100 at 5 GHz ?

**Comment by chrisk44** [comment]
What do you mean?

**Comment by schinfo** [comment]
Airodump shows channels >= 100 only as CH:-1 and not, for example, as CH:100 or CH:102

**Comment by chrisk44** [comment]
The latest version should do that. Please check.

**Comment by schinfo** [comment]
I've installed v1.5-beta.4 on my Nexus 5 - so it's the latest version.

**Comment by chrisk44** [comment]
Airodump doesn't always detect the channel. If it doesn't, it shows -1, otherwise you will see the correct channel. Since beta.2 the app shows the channel as reported by airodump-ng. If you run airodump-ng in a shell you will see it reporting channel -1. This usually (but not only) happens when the network is hidden.
